hiera.yaml

```diff
@@ -7,6 +7,10 @@ defaults:
 hierarchy:
     # It is acceptable for dot notation facts (eg. %{facts.some_fact}) to not exist
     # since they will simply return an empty string
+
+  - name: "local"
+    path: "/var/root/vault.yaml"
```

Comment:
If we copy `/var/root/vault.yaml` to the location on line 21/25 below in auto-puppet.sh (relative to the repo checkout) then we don't need to add anything.

Reply:
Ah this was looking at an old revision. Do you just place it during setup at this location?

Reply:
Ah, I see we drop an updated hiera.conf in auto-puppet.sh.